nautobot-app-ssot/{{ cookiecutter.project_slug }}/{{ cookiecutter.app_name }}/diffsync/models.py

```diff
@@ -0,0 +1,46 @@
+"""Diffsync models for {{ cookiecutter.app_name }}."""
+from typing import Optional, Annotated
+
+{% if cookiecutter.direction_of_sync == "From Nautobot" %}
+from diffsync import DiffSyncModel
+{% endif %}
+from nautobot.dcim.models import Device
+{% if cookiecutter.direction_of_sync == "To Nautobot" %}
+from nautobot_ssot.contrib import CustomFieldAnnotation, NautobotModel
+{% endif %}
+
+class DiffsyncDevice({{ "NautobotModel" if cookiecutter.direction_of_sync == "To Nautobot" else "DiffSyncModel" }}):
```

Comment:
Do we possibly want to change this to be DeviceModel instead? That's the naming pattern we use in the SSoT examples.

Reply:
The reason why I'm hesitant is because I noticed a lot of confusion between Django/Nautobot and Diffsync/SSoT models. This makes the distinction clear, whereas `DeviceModel` in my eyes doesn't. What do you think?

Reply:
Ok, fair. What about DeviceSSoTModel then? I just think it might make sense to differentiate a bit from DiffSync with some of the changes being discussed to SSoT.